.github/workflows/e2e.yaml

```diff
@@ -1,4 +1,4 @@
-name: e2e tests - containerd
```

Comment:
I want to know which is the container runtime used.

... at least until I can unify both by using the CRI-API instead of containerd / cri-api (for crio).

Reply:
Ha, I can't read. I thought "container? that sounds redundant!". I'll bring it back